src/site/content/en/blog/using-conversion-measurement/index.md

```diff
@@ -0,0 +1,207 @@
+---
+title: Using the Event Conversion Measurement API
+subhead: Must-dos and tips to use the Event Conversion Measurement API.
+authors:
+  - maudn
+hero: hero.jpg
+date: 2020-11-12
+updated: 2020-11-12
+tags:
+  - blog
+  - privacy
+---
+
+The [Event Conversion Measurement API](/conversion-measurement) measures when an ad click leads to a
+conversion, without using cross-site identifiers. Here, you'll find must-dos and tips to use this
+API locally or as an experiment for your end users.
+
+## Demo
+
+If you're considering using the API, see the
+[demo](https://goo.gle/demo-event-level-conversion-measurement-api) and the corresponding
+[code](https://github.com/GoogleChromeLabs/trust-safety-demo/tree/main/conversion-measurement) for a
+simple end-to-end implementation example.
+
+## Browser support
+
+The Event Conversion Measurement API can be supported:
+
+- As an [origin trial](/origin-trials/), from Chrome 86 beta and Chrome 87 (stable from mid-November
+  2020), to Chrome 88 (February 2021). Origin trials enable the API for **all visitors** of a given
+  [origin](/same-site-same-origin/#origin). **You need to register your origin for the origin trial
+  in order to try the API with end users**.
+- By turning on flags, in Chrome 86 and later. Flags enable the API on a **single user**'s browser.
+  **Flags are useful when [developing locally](#develop-locally)**.
+
+See details on the current status on the [Chrome feature
+entry](https://chromestatus.com/features/6412002824028160).
+
+### When is the API available?
+
+For the conversion measurement API to be available on a page, it needs to be:
+
+- Enabled on the **origin**.
+- AND—this is specific to this API—Enabled on the **browser**.
+
+With this, the API can be enabled as follows:
+
+<figure class="w-figure">
+  <img src="./api-enable.jpg" alt="Diagram: how to enable the API">
+</figure>
```

Comment:
This use of images seem borderline inaccessible to me. The way I think about is this: the `alt` description is the only way for people who use screen readers to get the information from the image. If the contents of the image are explained elsewhere in the page, then we don't have a problem. Because the people who use screen reader can get the information there. But if we're embedding information into diagrams, and that information is not available anywhere else on the page, then the people who use a screen reader are now at a disadvantage.

I said "borderline inaccessible" because I sense that this is basically just a summary of the information that you provide below. I just want to share my perspective on use of images and accessibility.

Reply:
Thanks for merging and for pointing this out. 
Yes, the diagrams are indeed a sum-up of the article's info, but I think exactly for this reason their content *should* be made accessible to screen readers as well. 
I'll open a patch PR for this and will make sure to address this in future articles!